core/templates/dev/head/components/entity-creation-services/topic-creation.service.ts.ts

```diff
@@ -75,6 +75,8 @@ angular.module('oppia').factory('TopicCreationService', [
             }, function() {
               $rootScope.loadingMessage = '';
             });
+        }, function() {
+
```

Comment:
Here and elsewhere, please add a comment

Reply:
Done added